mmdet/models/detectors/single_stage_instance_seg.py

```diff
@@ -95,7 +95,7 @@ def _forward(self,
             mask_outs = self.mask_head.forward(x)
         else:
             mask_outs = self.mask_head.forward(x, positive_infos)
-        outs = outs + (mask_outs, )
+        outs = outs + (mask_outs[0], )
```

Comment:
Can we add some comments above this line to explain? I do not quite understand mask_outs[0]

Reply:
Sorry, I find different torch vision will lead different error. In torch 1.7.1, JIT  don't support type NoneType, but in yolact, it's segm_preds is None, so use mask_outs[0]. But in torch 1.10.0, the error is always exist, I need check it again.